README.md

```diff
@@ -1,11 +1,19 @@
-## System tests
+## What is system-tests?
 
-Workbench designed to run advanced tests (integration, smoke, functional, fuzzing and performance)
+A workbench designed to run advanced tests (integration, smoke, functional, fuzzing and performance) against our suite of dd-trace libraries.
+
+When making changes to dd-trace, you'll commonly need to run the unmerged changes against the system tests (to ensure the feature is up to spec). But because system tests are outside of dd-trace-xx repos, testing against unmerged changes is not so straightforward. Various approaches  to this "chicken or the egg" problem are detailed [here](https://github.com/DataDog/system-tests/blob/main/docs/execute/how-to-approach-changes.md).
 
 ## Requirements
 
 `bash`, `docker` and `python3.12`. More infos in the [documentation](https://github.com/DataDog/system-tests/blob/main/docs/execute/requirements.md)
 
+## Weblog vs Parametric
+
+System-tests contains two types of tests: "weblog" and "parametric." Weblog apps support "automatic instrumentation." Weblog tests are used to test tracing integrations, security products, profiling, dynamic instrumentation, and more. Parametric tests are designed to validate tracer and span interfaces. They should be used to test operations such as creating spans, setting tags, setting links, injecting/extracting http headers, getting tracer configurations, etc. Weblog tests are designed to test a wider set of scenarios, so when in doubt, use weblog.
```

Comment:
There are more of them.

And I think we should work a little bit more on how we present them, the original purpose is a little bit different. Let's zoom for that